packages/venia-ui/lib/components/CategoryList/categoryTile.js

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+import React, { useMemo } from 'react';
 import { arrayOf, string, shape } from 'prop-types';
 import { mergeClasses } from '../../classify';
-import { Link, resourceUrl } from '@magento/venia-drivers';
+import { Link } from '@magento/venia-drivers';
 import defaultClasses from './categoryTile.css';
 import { useCategoryTile } from '@magento/peregrine/lib/talons/CategoryList/useCategoryTile';
+import Image from '../Image';
+
+const IMAGE_WIDTH = 80;
+const IMAGE_SIZES = new Map();
+IMAGE_SIZES.set('small', IMAGE_WIDTH);
```

Comment:
They don't have to be, no. Since it was new API I suppose I was trying to use / show it in as many places as possible. With these `Map`s changing to arrays, I can change this too.